cpp/src/strings/find.cu

```diff
@@ -58,9 +58,9 @@ std::unique_ptr<column> find_fn( strings_column_view const& strings,
                                  rmm::mr::device_memory_resource* mr,
                                  cudaStream_t stream )
 {
-    CUDF_EXPECTS( target.is_valid() && target.size()>0, "Parameter target must not be empty.");
+    CUDF_EXPECTS( target.is_valid(), "Parameter target must be valid.");
     CUDF_EXPECTS( start >= 0, "Parameter start must be positive integer or zero.");
-    if( (stop) > 0 && (start >stop) )
+    if( (stop) > 0 && (start > stop) )
```

Comment:
```suggestion
    if( (stop > 0) && (start > stop) )
```
The parentheses don't seem necessary here, but maybe this would be more consistent.